frontend/src/metabase/visualizations/echarts/cartesian/model/index.ts

```diff
@@ -57,7 +57,6 @@ export const getCardsSeriesModels = (
     return getCardSeriesModels(
       cardDataset,
       cardColumns,
-      index,
```

Comment:
Not related to the fix, I just deleted this parameter since it wasn't being used anymore